<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorldSettings</class>
 <widget class="QWidget" name="WorldSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>World properties</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&amp;World name</string>
            </property>
            <property name="buddy">
             <cstring>lineEditWorldName</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>V-Engine .world &amp;file</string>
            </property>
            <property name="buddy">
             <cstring>lineEditWorldFile</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>&amp;Soundtrack options</string>
            </property>
            <property name="buddy">
             <cstring>radioButtonSimple</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Soundtrack &amp;ID (music.xml)</string>
            </property>
            <property name="buddy">
             <cstring>lineEditSoundtrackID</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLineEdit" name="lineEditWorldName"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditWorldFile"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QRadioButton" name="radioButtonSimple">
              <property name="text">
               <string>Simple Track</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButtonLayered">
              <property name="text">
               <string>Layered Track</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditSoundtrackID"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Reset|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditWorldName</tabstop>
  <tabstop>lineEditWorldFile</tabstop>
  <tabstop>radioButtonSimple</tabstop>
  <tabstop>radioButtonLayered</tabstop>
  <tabstop>lineEditSoundtrackID</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
